ff/src/biginteger/arithmetic.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) fn mac_discard(a: u64, b: u64, c: u64, carry: &mut u64) {
 }
 
 /// Compute the NAF (non-adjacent form) of num
-pub fn find_naf(num: &[u64]) -> Vec<i64> {
+pub fn find_naf(num: &[u64]) -> Vec<i8> {
```

Comment:
naf change LGTM, but a bit confused why were bundling it into this PR

Reply:
Got pulled in from the previous PR (#294)

Reply:
(I can push it into its own PR if that's better)

Reply:
For the time being I've left it in, since it's a small change.